Add backTitle & backIcon to PhotoBrowser's properties.

mokabyls opened the pull request:
This pull request allows you to configure the label of the back button and the icon on back button. If nothing is entered, the default values are kept. **Breaking Change**: the "backImage" property in the TopBar component has been renamed to "backIcon".

Commit:
feat: add "backTitle" to PhotoBrowser's properties.
feat: add "backIcon" to PhotoBrowser's properties who take in params a react node or any ImageSourcePropType.
bc: rename "backImage" to "backIcon" in TopBar component.

mokabyls commented:
ping @halilb. Any news about the review ?

halilb commented on lib/bar/TopBar.js:
```diff
@@ -19,29 +19,31 @@ export default class TopBar extends React.Component {
         <TouchableOpacity style={styles.backContainer} onPress={onBack}>
-          <Image source={backImage} />
-          {Platform.OS === 'ios' &&
+          {backIcon && React.isValidElement(backIcon) && backIcon}
+          {backIcon && !React.isValidElement(backIcon) && <Image source={backIcon} />}
```
We might introduce a new prop called `backElement` to keep this backwards compatible as well:

```suggestion
const useBackElement = !!backElement && React.isValidElement(backElement)

          {useBackElement && backElement}
          {!useBackElement && !!backImage && <Image source={backImage} />}
```

halilb reviewed (commented):
Thanks for the work @mokabyls! And I'm sorry for reviewing this late.

This looks good except the breaking change part. I commented about a solution we might use by introducing a new prop without removing the old one.

Commit:
feat: add "backElement" to PhotoBrowser's properties who take in params a react node or any ImageSourcePropType.
fix: rollback "backIcon" to "backImage" in TopBar component.
feat: add "backElement" property in TopBar component

mokabyls commented:
Hi @halilb,

sorry for the delay. Just made adjustments on the library.
Tell me if you're ok with this modifications.

halilb commented:
> Hi @halilb,
> 
> sorry for the delay. Just made adjustments on the library.
> Tell me if you're ok with this modifications.

Thank you @mokabyls. It looks good!
